webdriver-spec.html

```diff
@@ -5780,118 +5774,19 @@
  <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
   with arguments <var>body</var> and <var>arguments</var>.
 
- <li><p>Wait for <a>execute a function body</a> to return
-  or for <a>session script timeout</a> milliseconds to expire,
-  whichever happens sooner.
+ <li><p>Let <var>promise</var> be a new Promise.
 
- <li><p>If the previous step completed due to the <a>session script timeout</a> being reached,
-  return <a>error</a> with <a>error code</a> <a>script timeout</a>.
-
- <li><p>If <var>result</var> is an <a>error</a>, return result.
+ <li><p>Let <var>result</var> be the result of <a>execute a function body</a>.
```

Comment:
I think the intention is to execute the injected code inside the promise, and resolve the promise to return the result?

```js
  new Promise((resolve, reject) => {
    try {
      let fn = new Function(userScript);
      resolve(fn(...userArgs));
    } catch (ex) {
      reject(ex);
    }
  }).then(onSuccess, onFailure);
```